man/systemd-run.xml

```diff
@@ -151,6 +151,19 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--slice-inherit</option></term>
+        <term><option>--slice-inherit=</option></term>
+
+        <listitem><para>Make the new <filename>.service</filename> or <filename>.scope</filename> unit part
+        of the specified inherited slice, instead of <filename>system.slice</filename>.</para>
```

Comment:
in user mode the services are put in the root slice by default

Reply:
I copied the doc from --slice. Could you perhaps fix the --slice doc for --user mode? Cause I am not sure what's happening or supposed to happen.